MODULE.bazel

```diff
@@ -0,0 +1,83 @@
+module(
+    name = "aspect_rules_format",
+    version = "0.0.0",
+    compatibility_level = 1,
+    repo_name = "aspect_rules_format",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.2")
+bazel_dep(name = "aspect_rules_js", version = "1.32.6")
+
+npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
+npm.npm_translate_lock(
+    name = "aspect_rules_format_npm",
+    link_workspace = "aspect_rules_format",
+    pnpm_lock = "//:pnpm-lock.yaml",
+)
+use_repo(npm, "aspect_rules_format_npm")
+
+bazel_dep(name = "buildifier_prebuilt", version = "6.3.3")
+bazel_dep(name = "aspect_bazel_lib", version = "1.11.1")
+bazel_dep(name = "rules_python", version = "0.26.0")
+bazel_dep(name = "rules_java", version = "6.4.0")
+bazel_dep(name = "rules_go", version = "0.39.1", repo_name = "io_bazel_rules_go")
+
+go = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+go.download(
+    name = "go_sdk",
```

Comment:
This line causes an error when used from another repo:
`Error in fail: go_sdk.download: name must not be specified in non-root module aspect_rules_format`